### PR TITLE
Init Sentry at service side

### PR DESCRIPTION
**Pull request by sitozzz**
- [x] Add config to production app when self hosted instance will be deployed
- [x] Wrap main application with nextjs error handler

### CODE REVIEW

**Review by toplenboren — APPROVED** [review]

**Review by toplenboren — APPROVED** [review]

File: apps/condo/next.config.js [hunk 1]
```diff
@@ -104,4 +109,25 @@ module.exports = withTM(withLess(withCSS({
 
         return config
     },
+
 })))
+
+if (sentryConfig['client']) {
+    module.exports = withSentryConfig(
+        module.exports,
```
Comment (SavelevMatthew) [inline apps/condo/next.config.js:117]: a bit strange syntax here to use module as variable :)

Maybe something like this?
```
let config = ...
if (...) {
    config = wrapper(config)
}

module.exports = config
```

File: apps/condo/sentry.client.config.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
```
Comment (SavelevMatthew) [inline apps/condo/sentry.client.config.ts:4]: Is this feature condo-only?

File: packages/keystone/KSv5v6/v5/prepareKeystone.js [hunk 3]
```diff
@@ -152,10 +163,12 @@ function prepareKeystone ({ onConnect, extendKeystoneConfig, extendExpressApp, s
                     debug: IS_ENABLE_APOLLO_DEBUG,
                     introspection: IS_ENABLE_DANGEROUS_GRAPHQL_PLAYGROUND,
                     playground: IS_ENABLE_DANGEROUS_GRAPHQL_PLAYGROUND,
-                    plugins: [
-                        new ApolloRateLimitingPlugin(keystone),
-                        new GraphQLLoggerPlugin(),
-                    ],
+                    plugins: apolloPlugins,
+                    // plugins: [
```
Comment (SavelevMatthew) [inline packages/keystone/KSv5v6/v5/prepareKeystone.js:167]: // ?

File: apps/condo/sentry.client.config.ts (hunk 2 above)
Comment (sitozzz) [reply apps/condo/sentry.client.config.ts:4]: For now - yes

**Pull request merged**